docs/src/pages/docs/manifest.md

```diff
@@ -122,6 +122,20 @@ program = "dex.so"
 [[test.genesis]]
 address = "22Y43yTVxuUkoRKdm9thyRhQ3SdgQS7c7kB6UNCiaczD"
 program = "swap.so"
+upgradeable_program = true
```

Comment:
This looks incorrect based on the implementation here https://github.com/coral-xyz/anchor/blob/65032e28999637ee0862a448af710eb30ccedf05/cli/src/config.rs#L952-L959

```suggestion
upgradeable = true
```

Reply:
yes my bad, thank you for catching!  fixed